Poker.BE/Poker.BE.Domain/Game/Round.cs

```diff
@@ -8,7 +9,85 @@ namespace Poker.BE.Domain.Game
 {
     public class Round
     {
-        // TODO: complete - set team member to do this
 
+        #region Enums
+        public enum Move
+        {
+            check,
+            call,
+            bet,
+            fold,
+            raise,
+            allin
+        }
+        #endregion
+
+        #region fields
```

Comment:
please change to `Fields` - capital **F**. same to all regions.

### by the way:
I have a code snippets that i made that will help you a lot with coding - like all regions and stuff...:
[My Code Snippets.zip](https://github.com/WorkshopSE/TexasHoldemPoker.repo/files/1001233/My.Code.Snippets.zip)